.github/workflows/build.yml

```diff
@@ -234,3 +234,20 @@ jobs:
       - name: "Generate Vault API Path Coverage Report"
         run: |
           go run cmd/coverage/main.go -openapi-doc=./testdata/openapi.json
+          #
+  # This job is used as a requirement for the repo's branch protection setup.
```

Comment:
Curious about the purpose of this job. Is this enabling branch protection on `release/` branches?